src/social/SocialIcon.js

```diff
@@ -69,11 +69,14 @@ const SocialIcon = props => {
     small,
     style,
     title,
-    type,
+    name,
     underlayColor,
+    type,
     ...attributes
   } = props;
 
+  let Icon = getIconType(type || 'font-awesome');
```

Comment:
Use of using `type || 'font-awesome'`  add a default prop at the bottom setting `type` to 'font-awesome'

Reply:
I copied this line of code from `Icon` component, so if you want me to change it in `SocialIcon` maybe we should do it there too?

Reply:
Yea maybe, but not in this PR